en/advanced_features/traffic_avoidance_utm.md

```diff
@@ -0,0 +1,34 @@
+# Air Traffic Avoidance: MAVLink
+
+PX4 can use MAVLink messages to support simple air traffic avoidance in [missions](../flight_modes/mission.md).
+If a potential collision is detected, PX4 can *warn*, immediately [land](../flight_modes/land.md), or [return](../flight_modes/return.md) (depending on the value of [NAV_TRAFF_AVOID](#NAV_TRAFF_AVOID)).
+
+This implementation is based on the [ADS-B traffic avoidance](../advanced_features/traffic_avoidance_adsb.md). The `UTM_GLOBAL_POSITION` MAVLink
+message is treated exactly the same as an ADS-B message.
+
+
+## Configure Traffic Avoidance
+
+Configure the action when there is a potential collision using the parameter below:
+
+Parameter | Description
+--- | ---
+<span id="NAV_TRAFF_AVOID"></span>[NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID) | Enable traffic avoidance mode specify avoidance response. 0: Disable, 1: Warn only, 2: Return mode, 3: Land mode.
+
+
+## Implementation
+
+PX4 listens for valid `UTM_GLOBAL_POSITION` MAVLink messages during missions. When one is received, it is converted
```

Comment:
Very FYI only, I usually layout each sentence on its own line. 
- Gitbook doesn't care - it joins the following line. 
- It doesn't make too much difference for git edit/review, but in some cases it can make the strings that are presented to translators "more logical".